src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/IProjectHostProvider.cs

```diff
@@ -12,7 +12,8 @@ internal interface IProjectHostProvider
         /// </summary>
         /// <param name="unconfiguredProjectHostObject">Host object for the underlying unconfigured project.</param>
         /// <param name="projectDisplayName">Project display name for the configured cross-targeting project.</param>
-        IConfiguredProjectHostObject GetConfiguredProjectHostObject(IUnconfiguredProjectHostObject unconfiguredProjectHostObject, string projectDisplayName);
+        /// <param name="targetFramework">Target framework for the configured cross-targeting project.</param>
+        IConfiguredProjectHostObject GetConfiguredProjectHostObject(IUnconfiguredProjectHostObject unconfiguredProjectHostObject, string projectDisplayName, string targetFramework);
```

Comment:
Why is this used? This is making the assumption that target framework is the thing that causes us to fire up language service "instances" - which I'm in the process of moving away from.